### ISSUE TITLE
CI on unrelated PR fails with an error on episode 15

### ISSUE THREAD

**Issue by ericseppanen** [rustacean-station/rustacean-station.org#58]
When queuing up a transcript for episode 008 (#57), I get a CI error:

```text
015-twir-339.md: 7116560b (reported) != 5267717b (actual)
```

This seems a little weird-- why don't I see this error on PR #55?

@nellshamrell any ideas?  It seems to behave as though commit 92154b7db4 is backwards or not applied, or we're serving the wrong mp3 file?

**Comment by jonhoo** [comment]
This is so weird. It looks like the file size of that file keeps fluctuating? As far as I can tell, the correct size is `5267717`. But that was rejected in #55. I have no idea what's going on with that... Maybe the CI script is wrong?

**Comment by jonhoo** [comment]
I pushed a fix for that now. Let's see what CI does.

**Comment by jonhoo** [comment]
Err, that seems to have passed... So strange..

**Comment by ericseppanen** [comment]
This one seems wrong as well, but isn't throwing errors?
```text
018-twir-343.md:length: "124"
```

This is roughly what you get if there's a 404 trying to fetch the mp3 file.  Note that curl doesn't return an error when this happens.  (Maybe `curl -f` would do better?)

**Comment by ericseppanen** [comment]
Ah, the episode 18 glitch is caused by there being a space after the URL.  Curl is trying to fetch that and getting a 404.

I'll play with the script a little bit to see if I can get it to fail in a less obscure way on 404 errors.

**Comment by jonhoo** [comment]
Ah, yes, that would certainly do it! Thanks.

**Comment by ericseppanen** [comment]
Unfortunately, I don't think commit 78b9555 explains why the file size of episode 15 would have been fluctuating.  As far as I can tell episode 18 had a completely separate glitch.

**Comment by jonhoo** [comment]
Yeah, that's true.. I guess I'll re-open in case it shows up again.

**Comment by ericseppanen** [comment]
I wonder if there's some kind of intrusive caching layer within github, so if your CI scripts run a lot and download big things, you're not wasting a ton of their bandwidth.

Now if that URL has changed over time (which I think did happen, as there was some discussion about audio quality on this episode), then it seems plausible that github actions might report nondeterministic file size fluctuations.

**Comment by jonhoo** [comment]
Yeah, I also think that URL may have changed once, and suspect that some kind of caching is involved. But hard to figure out without observing it reproducably..
